src/sentry/profiles/device.py

```diff
@@ -0,0 +1,440 @@
+    if model in ("iPad7,1", "iPad7,2"):
+        return "iPad Pro (12.9-inch, 2nd gen)"
+    if model in ("iPad7,3", "iPad7,4"):
+        return "iPad Pro (10.5-inch)"
+    if model in ("iPad8,1", "iPad8,2", "iPad8,3", "iPad8,4"):
+        return "iPad Pro (11-inch, 1st gen)"
+    if model in ("iPad8,5", "iPad8,6", "iPad8,7", "iPad8,8"):
+        return "iPad Pro (12.9-inch, 3rd gen)"
+    if model in ("iPad8,9", "iPad8,10"):
+        return "iPad Pro (11-inch, 2nd gen)"
+    if model in ("iPad8,11", "iPad8,12"):
+        return "iPad Pro (12.9-inch, 4th gen)"
+    if model in ("iPad13,4", "iPad13,5", "iPad13,6", "iPad13,7"):
+        return "iPad Pro (11-inch, 3rd gen)"
+    if model in ("iPad13,8", "iPad13,9", "iPad13,10", "iPad13,11"):
+        return "iPad Pro (12.9-inch, 5th gen)"
+    if model in ("iPad2,5", "iPad2,6", "iPad2,7"):
+        return "iPad mini (1st gen)"
+    if model in ("iPad4,4", "iPad4,5", "iPad4,6"):
+        return "iPad mini 2"
+    if model in ("iPad4,7", "iPad4,8", "iPad4,9"):
+        return "iPad mini 3"
+    if model in ("iPad5,1", "iPad5,2"):
+        return "iPad mini 4"
+    if model in ("iPad11,1", "iPad11,2"):
+        return "iPad mini (5th gen)"
+    if model in ("iPad14,1", "iPad14,2"):
+        return "iPad mini (6th gen)"
+    if model in ("Watch1,1", "Watch1,2"):
+        return "Apple Watch (1st gen)"
+    if model in ("Watch2,6", "Watch2,7"):
+        return "Apple Watch Series 1"
+    if model in ("Watch2,3", "Watch2,4"):
+        return "Apple Watch Series 2"
+    if model in ("Watch3,1", "Watch3,2", "Watch3,3", "Watch3,4"):
+        return "Apple Watch Series 3"
+    if model in ("Watch4,1", "Watch4,2", "Watch4,3", "Watch4,4"):
+        return "Apple Watch Series 4"
+    if model in ("Watch5,1", "Watch5,2", "Watch5,3", "Watch5,4"):
+        return "Apple Watch Series 5"
+    if model in ("Watch5,9", "Watch5,10", "Watch5,11", "Watch5,12"):
+        return "Apple Watch SE"
+    if model in ("Watch6,3", "Watch6,4"):
+        return "Apple Watch Series 6"
+    if model in ("AppleTV1,1"):
+        return "Apple TV (1st gen)"
+    if model in ("AppleTV2,1"):
+        return "Apple TV (2nd gen)"
+    if model in ("AppleTV3,1", "AppleTV3,2"):
+        return "Apple TV (3rd gen)"
+    if model in ("AppleTV5,3"):
+        return "Apple TV (4th gen)"
+    if model in ("AppleTV6,2"):
+        return "Apple TV 4K"
+    if model in ("AppleTV11,1"):
+        return "Apple TV 4K (2nd gen)"
+    if model in ("i386"):
+        return "iOS Simulator (i386)"
+    if model in ("x86_64"):
+        return "iOS Simulator (x86_64)"
+    if model.startswith("iPhone"):
+        return "Unknown iPhone"
+    if model.startswith("iPad"):
+        return "Unknown iPad"
+    if model.startswith("AppleTV"):
+        return "Unknown Apple TV"
+    if model.startswith("Watch"):
+        return "Unknown Apple Watch"
+    return "Unknown iOS Device"
+
+
+def ios_cpu_core_max_frequencies_mhz(model: str) -> List[int]:
```

Comment:
This should be `Sequence[int]` or `Tuple[int]`. 

It seems like sometimes you also return bare ints. Should we make this consistent?

Reply:
I fixed the bare ints, it should really be `Tuple[int]`.